src/server.cpp

```diff
@@ -893,6 +893,8 @@ void Server::AsyncRunStep(bool initial_step)
 		// We'll log the amount of each
 		Profiler prof;
 
+		std::list<v3s16> node_meta_updates;
```

Comment:
Can you use a vector instead? You can reserve the capacity with the size of `m_unsent_map_edit_queue`, it is guaranteed to stay below, so its only one allocation for the whole vector.